operator/roles/forkliftcontroller/templates/populator/deployment-populator.yml.j2

```
@@ -0,0 +1,27 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ populator_controller_deployment_name }}
+  namespace: {{ app_namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ app_name }}
+  template:
+    metadata:
+      labels:
+        app: {{ app_name }}
+    spec:
+      serviceAccountName: forklift-controller
+      containers:
+        - name: {{ populator_controller_container_name }}
+          image: {{ populator_controller_image_fqin }}
+          imagePullPolicy: Always
+          args:
+            - --image-name={{ populator_ovirt_image_fqin }}
```

should be 
```suggestion
            - --image-name={{ populator_controller_image_fqin }}
```
right?

---

nope, in the args we will send the populator image to use (ovirt populator). in the future - with OSP. we will need 2 args here (e.g. `--ovirt-image-name` & `--osp-image-name`).

---

ack, it was confusing because it's different than the convention of passing-in those configurable image urls using environment variables - I change it to comply with this convention